docs/application/native/guides/app-management/ui-app/efl-ui-app.md

```diff
@@ -61,7 +61,7 @@ The following figure illustrates the application state transitions.
 
```

Comment:
On line 42, `[application states](applications.md#state_change)` is broken. It looks like `(../applications.md#state_change)` is correct.

Reply:
done